domain/domain.py

```
@@ -298,7 +294,7 @@ def get_random_domain(self, attr, cur_value):
         if random.random() > self.sampling_prob:
             return []
         domain_pool = set(self.single_stats[attr].index.astype(str))
-        domain_pool.remove(cur_value)
+        domain_pool.discard(cur_value)
```

what's the difference between `remove` and `discard`?

---

Remove throws a `ValueError` if the value is not in the set, `discard` does not. This actually came up when I changed up some of the exception handlers to not silence exceptions as "status" messages.